polaris-tokens/src/token-groups/shadow.ts

```diff
@@ -78,7 +78,7 @@ export const shadow: {
   },
   'shadow-card-md-experimental': {
     value:
-      '0px 3px 1px -1px rgba(0, 0, 0, 0.07), inset 0px -1px 0px rgba(0, 0, 0, 0.16), inset 1px 0px 0px rgba(0, 0, 0, 0.1), inset -1px 0px 0px rgba(0, 0, 0, 0.1), inset 0px 1px 0px rgba(255, 255, 255, 0.3)',
+      '0 3 1 -1 rgba(0, 0, 0, 0.07)  0 -1 0 0 rgba(0, 0, 0, 0.16)  1 0 0 0 rgba(227, 227, 227, 1)  -1 0 0 0 rgba(227, 227, 227, 1)  0 1 0 0 rgba(227, 227, 227, 1)',
```

Comment:
Looks like all the `px` units and commas were stripped from the value. This intentional?

Reply:
thnx for the catch 🙏  fixed

Reply:
Nice! I'm still seeing unitless lengths e.g. `0 3 1 -1` instead of `0 3px 1px -1px`. Intentional?